src/main/java/com/faforever/gw/services/messaging/lobby_server/LobbyService.java

```diff
@@ -172,8 +173,8 @@ private void enqueue(OutgoingLobbyMessage message) {
         }
     }
 
-    public void createGame(UUID battleId) {
-        enqueue(new CreateGameMessage(UUID.randomUUID(), battleId));
+    public void createGame(UUID battleId, List<Long> participants) {
+        enqueue(new CreateGameMessage(UUID.randomUUID(), battleId, participants));
```

Comment:
In the long run we'll need a whole GameOptions object which also contains the mapVersion, the teams, a list of mods, reinforcements etc.

But for now it will do.